grades/models.py

```diff
@@ -41,6 +58,8 @@ class FinalGrade(TimestampedModel, AuditableModel):
     )
     course_run_paid_on_edx = models.BooleanField(default=False)
 
+    objects = FinalGradeQuerySet.as_manager()
```

Comment:
this should be at the very beginning of the class definition with a pretty detailed explanation of what it is doing, given that this is the very first time that we use a custom manager